### ISSUE TITLE
RCT1 Scenario Menu doesn't display completion

### ISSUE THREAD

**Issue by wordy00** [OpenRCT2/OpenRCT2#4951]
**OS:** Windows 10 Education 64-bit
**Version:** v0.0.6
**Commit/Build:** develop build 99a662f

I completed an RCT1 scenario imported into RCT2 Open but it doesn't display the completion on the scenario select menu. 

- [x] Reproducible in RCT2 (vanilla). Working in Vanilla

**Comment by Gymnasiast** [comment]
That's right, as it needs to set the scenario path in order to do so.

**Comment by wordy00** [comment]
Thanks for the reply. Is there any way I can do this ?

**Comment by Gymnasiast** [comment]
Unfortunately not. We need to adjust the game to set the scenario path properly when importing RCT1 scenarios, or come up with another way to keep progress of RCT1 scenarios.

**Comment by IntelOrca** [comment]
I can't reproduce this, if I open Forest Frontiers then use the cheats to complete the scenario - I am able to enter my name and then see it on the scenario list.

**Comment by Gymnasiast** [comment]
Did you open Forest Frontiers from an SV4 or SC4?

**Comment by IntelOrca** [comment]
Oh are we talking about SC4? In that case, does those even show on the scenario list?

**Comment by Gymnasiast** [comment]
No. (They should, eventually, but they don't show up currently.)

**Comment by IntelOrca** [comment]
OK - I was just confused then, if it doesn't show up to play it then obviously it can't show whether its completed or not.

**Comment by wordy00** [comment]
It is a SC6 file. It shows on the scenario list and allows me to load it up. I will try and shed more details on the issue.

- This screenshot shows how the scenario list looks on my game. https://gyazo.com/7884ab08b0ad402880d62c26c0f76cff

- This screenshot shows the objective completion screen that I completed legitimately without cheats. It does not allow me to enter my name. https://gyazo.com/cd09a59797df5982e4943916dbae8b19

- This screenshot shows me using cheats to complete the scenario. It allows me to enter my name this time. https://gyazo.com/01124faffa23b72c5617bad99d14a4af

Thanks for the help.

**Comment by IntelOrca** [comment]
I think this is caused by opening saved games, or saved games not saving the scenario filename. I will look into it.

**Comment by IntelOrca** [comment]
@wordy00 It should now be fixed in the next version.

**Comment by wordy00** [comment]
Thanks for the help. Much appreciated.

**Comment by wordy00** [comment]
Hi,

I can confirm the game allows me to enter my name when the scenario has been completed. I simulated the park rating and guests using cheats and increased the game speed to maximum. After I have entered my name it displays question marks on the scenario screen instead of my name. It correctly displays company value though. 

Screenshot showing the error: https://gyazo.com/5e216761e9d1efb862328bc6e9eddf64

Not a game breaking bug by any means but just thought I'd let you know :). Again, thanks for all the help.
